rose/server/score_test.py

```diff
@@ -273,7 +272,9 @@ def setup_method(self, m):
         self.player2 = player.Player("B", car=0, lane=1)
 
     def process(self):
-        players = {self.player1.name: self.player1, self.player2.name: self.player2}
+        players = {
+            self.player1.name: self.player1,
+            self.player2.name: self.player2}
```

Comment:
This works, but better format it like this:

    players = {self.player1.name: self.player1,
               self.player2.name: self.player2}